ext/user_config/main.php

```diff
@@ -141,7 +141,7 @@ public function onPageRequest(PageRequestEvent $event)
                         return;
                     }
 
-                    $uobe = new UserOptionsBuildingEvent($display_user, new SetupPanel($user_config));
+                    $uobe = new UserOptionsBuildingEvent($display_user, new SetupPanel($user_config, new SetupTheme));
```

Comment:
This is the only thing that's not working with the `SetupBlock` changes. How should I access `SetupTheme`? Maybe `UserConfigTheme` should extend `SetupTheme` now? I feel like it should.